test/services/TemplatingService.spec.ts

```diff
@@ -0,0 +1,110 @@
+import Mail from '@/interfaces/Mail';
+import UserRessource from '@/interfaces/UserRessource';
+import TemplatingService from '@/services/TemplatingService';
+import message from '@test/data/message';
+import { AssertionError } from 'assert';
+import { expect } from 'chai';
+import { messaging as firebaseMessaging } from 'firebase-admin';
+import 'mocha';
+
+// Instantiate the service
+const templatingService =
+  new TemplatingService(message.platform, message.template, message.payload, message.languagePayloads);
+const receiver: UserRessource = (message.receivers[0] as UserRessource);
+
+describe('TemplatingService.createMailMessage', () => {
+
+  let mail: Mail;
+
+  beforeEach('create a mail message.', () => {
+    mail = templatingService.createMailMessage(receiver);
+  });
+
+  it('should create a valid e-mail.', () => {
+
+    expect(mail)
+      .to.have.keys('from', 'to', 'subject', 'text', 'html');
+
+    expect(mail.to)
+      .to.not.to.have.lengthOf(0);
+
+    expect(mail.subject)
+      .to.not.to.have.lengthOf(0);
+
+    expect(mail.text)
+      .to.not.to.have.lengthOf(0);
+
+    expect(mail.html)
+      .to.not.to.have.lengthOf(0);
+  });
```

Comment:
I would check that we have the expected length (or value) and not that the string is not empty.